atomicfu-gradle-plugin/src/test/resources/projects/jvm-simple/build.gradle

```diff
@@ -5,6 +5,10 @@
 apply plugin: 'kotlinx-atomicfu'
 apply plugin: 'kotlin'
 
+kotlin.target.compilations.all {
+    kotlinOptions.freeCompilerArgs += '-Xallow-jvm-ir-dependencies'
```

Comment:
This flag allows to use IR dependencies with projects being compiled using old JVM compiler. As far as I know, there is no other side effects

Reply:
Yeah, this flag has no effect other than allow old backend to load binaries generated by new backend.

There's an inverse option, `-Xir-binary-with-stable-abi`, which is used with the new backend and makes the resulting binaries "stable", so that both old and new backends would load them fine. So it can be enabled right where `useIR` is enabled. I'd actually slightly prefer it since it's more local, but both variants are OK for me.

(Also note that we're going to mark all binaries generated by JVM IR as stable since 1.4.30, so these flags won't be necessary anymore.)